Classes/Controller/PluginController.php

```diff
@@ -587,39 +585,40 @@ public function getMap()
                         )
                     ),
                 )
-                ->add('orderBy', 'FIELD(uid, ' . implode(',', $this->config['layer']) . ')', true)
+                #->add('orderBy', 'FIELD(uid, ' . implode(',', $this->config['layer']) . ')', true)
```

Comment:
Does this break some functionality?

Reply:
oh, well spotted. That only breaks in sqlite as sqlite does not support `FIELD`. I added this while creating tests. Will revert.